### ISSUE TITLE
Hyperparameter Summary

### ISSUE THREAD

**Issue by abondre98** [abondre98/Adaptive-Global-LCA-Advisor-with-Dynamic-EF-Retrieval#133]
○	Rank, learning rate, batch size, number of epochs, final best model step/epoch.
○	Note any changes made mid-training (e.g., lowered LR after epoch 2).
